acceptance/acceptance_test.go

```diff
@@ -3315,6 +3364,157 @@ func createStackImage(dockerCli client.CommonAPIClient, repoName string, dir str
 	}))
 }
 
+func createFlattenBuilder(
```

Comment:
I didn't want to invest a lot of time refactoring the setup for a builder creation, this method is very similar to create complex builder, maybe it is time to continue Javier's work on refactoring the tests